addons/account/models/account.py

```diff
@@ -1375,29 +1375,40 @@ def _check_repartition_lines(self, lines):
         base_line = lines.filtered(lambda x: x.repartition_type == 'base')
         if len(base_line) != 1:
             raise ValidationError(_("Invoice and credit note repartition should each contain exactly one line for the base."))
+        tax_lines = lines - base_line
+        if not tax_lines:
+            if base_line.factor_percent == 100:
```

Comment:
- Why do you check this condition particularly?
- isn't it always true?

Reply:
If it isn't true, then it means that there are missing tax lines (the tax lines are not optional and are used)

Reply:
I mean this one:
`if base_line.factor_percent == 100:`